app/views/bets/show.html.erb

```diff
@@ -1,49 +1,41 @@
-<div class="container">
-  <hr>
-  <div class="row">
-    <div class="col col-lg-12">
-      <h2>Viewing bet: <%= @bet.id %></h2>
+<%= render 'shared/bet_display', bet: @bet %>
+
+<hr>
+<div class="row">
+  <div class="col col-lg-6 offset-lg-3">
+    <div class="card">
+      <div class="card-header text-muted">
+        <div class="row">
+          <div class="col col-lg-4">User</div>
+          <div class="col col-lg-4">Option ID</div>
+          <div class="col col-lg-4">Amount</div>
+        </div>
+      </div>
+      <div class="card-block">
+        <table class="table table-hover">
+          <%= render @bet.user_bets %>
+        </table>
+      </div>
     </div>
   </div>
-  <hr>
```

Comment:
This section has been deleted in one of the previous PR's already, so it should get resolved upon merging.